docker/generate_build_matrix.py

```diff
@@ -165,8 +166,8 @@ def _write_table(table_tag: str, table_contents: str):
 
 
 def _main():
-    python_versions = ['3.10']
-    pytorch_versions = ['2.1.2', '2.0.1', '1.13.1']
+    python_versions = ['3.11', '3.10']
```

Comment:
this builds 3.11 for all images which is kinda slow. can we only build for 2.2.0?

Reply:
fixed